sssom/context.py

```diff
@@ -26,6 +29,11 @@ def get_jsonld_context():
 
     :return: JSON-LD context
     """
+    schema_path = pkg_resources.resource_filename(
```

Comment:
Does this work when installing with pip? Do we have a precedence in the code for this (loading the resource this way I mean). I feel `schema_path` should be in constants.py somehow, but up to you.

Reply:
I forgot that schema_path was already implemented in `constants.py`. Updating code accordingly. The variable name is `SCHEMA_YAML` and is used in many parts of the project. So it definitely works since previous releases did not have any issues when we implemented `sssom-schema`.